apps/sms/views/conversation/js/conversation.js

```diff
@@ -707,6 +696,15 @@ var ConversationView = {
     if (Navigation.isCurrentPanel('thread-list')) {
       // We don't want to clean these things when moving from composer to
       // conversation
+      // Revoke thumbnail URL for every image attachment rendered within thread
+      // Executed only when moving out of a conversation.
+      var nodes = this.container.querySelectorAll(
```

Comment:
__question:__ mmm does that mean we won't revoke any URL if we navigate from Conversation to NewMessage or another Conversation (e.g. via "Send message" link)? Wondering if we can use ```isConversationPanel``` here instead